### ISSUE TITLE
In react native debugger-ui, when reloading the APP, sometimes the screen will be loaded twice

### ISSUE THREAD

**Issue by lyswhut** [wix/react-native-navigation#7065]
## 🐛 Bug Report

In react native debugger-ui, when reloading the APP, sometimes the screen will be loaded twice (the first time is executed before the registerAppLaunchedListener event is issued)

### Have you read the [Contributing Guidelines on issues](https://wix.github.io/react-native-navigation/docs/meta-contributing/)?

Yes it is

## To Reproduce

1. Bash run:
```bash
react-native init test --version 0.63.4
cd test
yarn add react-native-navigation
npx rnn-link
```
2. Update index.js file
```js
import {Navigation} from "react-native-navigation";
import App from "./App";
Navigation.registerComponent('com.myApp.WelcomeScreen', () => App);
Navigation.events().registerAppLaunchedListener(() => {
  console.log('registerAppLaunchedListener')
  Navigation.setRoot({
     root: {
       stack: {
         children: [
           {
             component: {
               name:'com.myApp.WelcomeScreen'
             }
           }
         ]
       }
     }
  });
});
```
3. Run `npm run android`
4. Open the developer menu and select "Debug", which will automatically open the browser and open react native debugger-ui in the browser
5. Open the developer tools of the browser, and then click the "reload" button on the page, try "reload" several times to see the problem

## Expected behavior

Click the "reload" button to load the page only once

## Actual Behavior

Sometimes the reload page is loaded twice (once loaded before the registerAppLaunchedListener event is issued, and the other time is loaded after the registerAppLaunchedListener event is issued)
![image](https://user-images.githubusercontent.com/20365169/112716263-425c4200-8f20-11eb-9a6b-b227d1b10151.png)

## Your Environment

- React Native Navigation version: 7.11.3
- React Native version: 0.63.4
- Platform(s) (iOS, Android, or both?): Android (not tested on other platforms)
- Device info (Simulator/Device? OS version? Debug/Release?): Simulator (Android 7.1.4) and Device (Android 10), Debug mode

## Reproducible Demo

Since the steps to reproduce this problem are relatively simple, I have not created a repo now, but if you really need it, I can create it later

**Comment by boyuanjiang0-0** [comment]
I am facing the problem now, it drive me crazy

**Comment by karimb11** [comment]
Issue is still occurring in latest version of react-native and react-native-navigation as of this comment.

**Comment by yedidyak** [comment]
Closing this as part of a maintainer cleanup of open issues older than one year.

Decision: Closed: stale/non-critical; no evidence it is a critical current-version RNN issue.

If this still reproduces on the latest React Native Navigation with a minimal reproduction, please open a new issue with current RN/RNN versions and repro steps.
